README.md

```diff
@@ -1,51 +1,80 @@
 
 ## Usage
 
 ### 1. Pull the Docker image
 
-Use the following docker command to run your container
-
 Donwload the logzio/webdav-fetcher image:
 
 ```sh
-docker pull logzio/webdav-fetcher:latest
+docker pull logzio/sfcc-logs-fetcher:latest
 ```
 
 ### 2. Run the container
 
-Use the following docker command to run your container.
+Use the following Docker command to run your container.
 
 ```sh
 docker run -d -e LOGZIO_SHIPPING_TOKEN=<logzio_shipping_token>  \
 -e LOGZIO_LISTENER_URL=<logzio_listener_url> \
 -e SFCC_HOSTNAME=<your_sfcc_host> \
 -e SFCC_CLIENT_ID=<your_sfcc_client_id> \
 -e SFCC_CLIENT_SECRET=<your_sfcc_client_secret> \
-logzio/webdav-fetcher:latest
+logzio/sfcc-logs-fetcher:latest
 ```
 
 | Parameter             | Description                                                                                                                                                                                                                                                                                                                                   | Required |
 | --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------: |
 | LOGZIO_LISTENER_URL   | Determines protocol, listener host, and port. For example, https://<<LOGZIO_LISTENER_URL>>:8071.Replace <<LOGZIO_LISTENER_URL>> with your region's listener host (for example, listener.logz.io). For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html) . |      Yes |
 | LOGZIO_SHIPPING_TOKEN | The [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.                                                                                                                                                                                                                                             |      Yes |
-| SFCC_HOSTNAME         | Hostname from what host need to send logs (ex. dev01-mysandbox.demandware.net)                                                                                                                                                                                                                                                                |      Yes |
+| SFCC_HOSTNAME         | Hostname from the host that need to send logs from (ex. dev01-mysandbox.demandware.net)                                                                                                                                                                                                                                                       |      Yes |
```

Comment:
I think it should be SFCC_SERVER_NAME and then the description will be SFFC Server name from where you would like to collect logs